config/locales/ar.yml

```diff
@@ -0,0 +1,386 @@
+ar:
```

Comment:
It would be best if you translated our texts instead of putting example text and sending verification to Steve